main.go

```diff
@@ -73,86 +81,99 @@ var (
 	defaultKeepAliveDuration = constants.DefaultKeepAliveDuration
 )
 
-var namespace string
-
 // InitFlags initializes the flags.
 func InitFlags(fs *pflag.FlagSet) {
-	logsv1.AddFlags(logOptions, fs)
+	// Flags specific to CAPV
 
-	flag.StringVar(
-		&managerOpts.MetricsBindAddress,
-		"metrics-bind-addr",
-		"localhost:8080",
-		"The address the metric endpoint binds to.")
-	flag.BoolVar(
-		&managerOpts.LeaderElection,
-		"leader-elect",
-		true,
-		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(
+	fs.StringVar(
 		&managerOpts.LeaderElectionID,
 		"leader-election-id",
 		defaultLeaderElectionID,
 		"Name of the config map to use as the locking resource when configuring leader election.")
-	flag.StringVar(
-		&namespace,
-		"namespace",
-		"",
-		"Namespace that the controller watches to reconcile cluster-api objects. If unspecified, the controller watches for cluster-api objects across all namespaces.")
-	flag.StringVar(
-		&profilerAddress,
-		"profiler-address",
-		defaultProfilerAddr,
-		"Bind address to expose the pprof profiler (e.g. localhost:6060)")
-	flag.DurationVar(
-		&syncPeriod,
-		"sync-period",
-		defaultSyncPeriod,
-		"The interval at which cluster-api objects are synchronized")
+
 	flag.IntVar(
```

Comment:
we moved all other vars to use fs. this one is still using flag (?)

Reply:
jup, missed that one 👍